src/Illuminate/Foundation/Auth/ThrottlesLogins.php

```diff
@@ -53,7 +53,10 @@ protected function sendLockoutResponse(Request $request)
         );
 
         throw ValidationException::withMessages([
-            $this->username() => [Lang::get('auth.throttle', ['seconds' => $seconds])],
+            $this->username() => [Lang::get('auth.throttle', [
+                'seconds' => $seconds,
+                'minutes' => ceil($seconds * 60),
```

Comment:
Am I reading/understanding this wrong, should this not be `'minutes' => ceil($seconds / 60),` ?

Reply:
Yep, you're right. I already closed my project so I thought I would re-type this in GitHub. I will change this immediately.

Reply:
See #30017.